Assets/Scripts/Models/FurnitureActions.cs

```diff
@@ -26,6 +26,11 @@ public FurnitureActions()
         LuaUtilities.RegisterGlobal(typeof(World));
     }
 
+    public static bool IsInitialized
+    {
+        get{ return _Instance != null; }
```

Comment:
The variable `_Instance` doesn't exist and the line should have the parenthesis on new lines.